### PR TITLE
feature/HuggingFace

### PR DESCRIPTION
**Pull request by chungyau97**
step 1 and 2 are not optional

### CODE REVIEW

File: chat-models/google-vertexai.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 <figure><img src="../.gitbook/assets/gcp_credential/vertex_AI_enable.png" alt="" width="563"><figcaption></figcaption></figure>
 
-## Create credential file (Optional)
+## Create credential file _(Optional)_
```
Comment (HenryHengZJ) [inline chat-models/google-vertexai.md:16]: why the change of this?

Comment (chungyau97) [reply chat-models/google-vertexai.md:16]: standardize the way of writing [OPTIONAL] and (Optional) to become _(Optional)_
